pkg/storage/utils.go

```diff
@@ -0,0 +1,20 @@
+package storage
+
+import (
+	"strings"
+)
+
+func matchesNameOrID(filter, ID string, names []string) bool {
+	if filter == ID {
+		return true
+	}
+	for _, name := range names {
+		if filter == "" || filter == name {
+			return true
+		}
+	}
+	if len(filter) < 7 {
```

Comment:
It's a placeholder value, chosen mainly because that's the length that container and image IDs are often truncated for display by dockerd.  A longer version would probably use a truncindex, since all we're really looking for is a sufficiently-long value to be unambiguous about which image or container or pod the caller is referring to.  I'll see about switching to doing that.